src/builders.ts

```diff
@@ -275,17 +304,27 @@ export async function makeSTXTokenTransfer(
 
   const payload = createTokenTransferPayload(options.recipient, options.amount, options.memo);
 
-  const addressHashMode = AddressHashMode.SerializeP2PKH;
-  const privKey = createStacksPrivateKey(options.senderKey);
-  const pubKey = getPublicKey(privKey);
   let authorization = null;
-
-  const spendingCondition = new SingleSigSpendingCondition(
-    addressHashMode,
-    publicKeyToString(pubKey),
-    options.nonce,
-    options.fee
-  );
+  let spendingCondition = null;
+
+  if ('publicKey' in options) {
+    // multi-sig
```

Comment:
this would not be multi-sig case, right?

Reply:
yup, this is a typo!